stubs/pywin32/_win32typing.pyi

```diff
@@ -8,6 +8,8 @@ from win32.lib.pywintypes import TimeType
 
 _TwoIntSequence: TypeAlias = Sequence[int]
 _FourIntSequence: TypeAlias = Sequence[int]
+# Is actually pywin.mfc.DocTemplate
+DocTemplate: TypeAlias = Incomplete
```

Comment:
This entire module is already private. Names that reflect real types from pywin32 don't have a leading `_` here.